modules/appeals_api/spec/requests/v2/notice_of_disagreements_controller_spec.rb

```diff
@@ -49,6 +49,36 @@ def base_path(path)
       end
     end
 
+    context 'returns 422 when birth date is not a date' do
+      let(:error_content) do
+        { 'status' => 422, 'detail' => "'apricot' did not match the defined pattern" }
+      end
+
+      it 'when given a string for the birth date ' do
+        @headers.merge!('X-VA-Birth-Date' => 'apricot')
+        post(path, params: @data.to_json, headers: @headers)
+        expect(response.status).to eq(422)
+        expect(parsed['errors']).to be_an Array
+      end
+    end
+
+    context 'returns 422 when decison date is not a date' do
+      let(:error_content) do
+        { 'status' => 422, 'detail' => "'banana' did not fit within the defined length limits" }
```

Comment:
Hm. Seems like the detail should be something like, "'banana' is not a date", right?

Reply:
Note: We're going to create a ticket to look into whether this is the expected detail.